README.md

```diff
@@ -680,6 +680,8 @@ Listed below are all configuration options.
   Acceptable values - `utc`, `local` (to use system time zone) and time zone name e.g. `Eastern Time (US & Canada)`. Default is `local`
 * `store_datetime_as_string` - if `true` then Dynamoid stores :datetime fields in ISO 8601 string format. Default is `false`
 * `store_date_as_string` - if `true` then Dynamoid stores :date fields in ISO 8601 string format. Default is `false`
+* `backoff` - is a hash: key is a backoff strategy (symbol), value is parameters for the strategy. Is used in batch operations. Default id `nil`
+* `backoff_strategies`: is a hash and contains all available strategies. Default is { constant: ..., exponential: ...}
```

Comment:
Is this part of config so people can write and plugin their own backoff strategy and add it here?

Reply:
> Is this part of config so people can write and plugin their own backoff strategy and add it here?

Yep. I can imagine people would like to use another implementation of default `exponential` backoff, e.g. not binary and I wouldn't like to write universal exponential strategy.